yetanotherco/aligned_layer
# fix (batcher): gnark verifier bindings on linux

Gnark verifier bindings on linux were not working. 

This PR fixes those issues. It has been tested on linux.

# To test
1. Get a linux machine (old code worked on docker so docker does not count)
2. Start anvil, batcher, aggregator and operator
3. Run `make batcher_send_groth16_bn254_task`, `make batcher_send_plonk_bls12_381_task` and `make batcher_send_plonk_bn254_task`
4. Batcher should verify them correctly
